src/mysqlTreeDataProvider.ts

```diff
@@ -76,6 +77,19 @@ export class MySQLTreeDataProvider implements vscode.TreeDataProvider<INode> {
         AppInsightsClient.sendEvent("addConnection.end");
     }
 
+    public async keywordFilter(node: DatabaseNode | ConnectionNode) {
+        vscode.window.showInputBox({
+           prompt: "" , placeHolder: "keyword", value: node.keyword,
+           validateInput: (keyword: string) => {

```

Comment:
It this by design? It will trigger `this.refresh(node)` every time user input or delete a character. For the better practice, it should be only trigger when user press `Enter`